examples/instrumentation-quickstart/src/test/java/com/example/demo/DockerComposeTestsIT.java

```diff
@@ -81,7 +84,11 @@ public void testApp() throws InterruptedException, IOException, URISyntaxExcepti
   }
 
   private static String getenvNotNull(String key) {
-    return Preconditions.checkNotNull(
```

Comment:
I can alternatively add a guava dependency, wdyt?

Reply:
I think it's okay to leave this validation out instead of adding a dependency. The validation isn't key to what we're trying to demonstrate.